app/models/form_profiles/complaint_tool.rb

```diff
@@ -8,4 +8,17 @@ def metadata
       returnUrl: '/applicant-relationship'
     }
   end
+
+  def convert_country!(form_data)
+    country = form_data.try(:[], 'address').try(:[], 'country')
```

Comment:
Should we use `dig` here?  I honestly don't know if it makes any difference.

Reply:
We would need to use `try` if `form_data` can be `nil`

Reply:
i think if prefill is not enabled for a form then `form_data` can be nil